docs/group_concepts/stages_concept.rst

```diff
@@ -0,0 +1,31 @@
+Launch Stages Overview
```

Comment:
For all of these launch stages I will write the descriptions in Confluence and send you a link so that you can copy/paste here.